components/expenses/Expense.js

```diff
@@ -155,6 +156,11 @@ class Expense extends React.Component {
     mode = mode || view;
 
     const canPay = LoggedInUser && LoggedInUser.canPayExpense(expense) && expense.status === 'APPROVED';
+    const canReverseOtherPayment =
```

Comment:
I'm not sure about this variable name. Have you considered alternatives?

Reply:
`canMarkExpenseAsUnpaid` 🤔